wolfssl/wolfcrypt/settings.h

```diff
@@ -354,6 +356,7 @@
     #define SINGLE_THREADED
     #define NO_WOLFSSL_DIR
     #define WOLFSSL_HAVE_MIN
+    #define WOLFSSL_HAVE_MAX
```

Comment:
This is inside the WOLFSSL_UTASKER define.  As this is not related to VxWorks, did you test this with the uTasker OS and compiler?